protocol/src/main/java/org/opensearch/sql/protocol/response/format/JsonResponseFormatter.java

```diff
@@ -47,6 +47,10 @@ public String format(Throwable t) {
         (style == PRETTY) ? prettyFormat(t) : compactFormat(t));
   }
 
+  public String getFormat() {
+    return "application/json; charset=UTF-8";
```

Comment:
Needs a unit test.

Also, move this to a constant (public static final String)

Reply:
added unit test and made constant in https://github.com/Bit-Quill/opensearch-project-sql/pull/279/commits/0df2a1b34b182cea23bfb490d7143bd726aa00d4